massa-network-exports/src/settings.rs

```diff
@@ -52,6 +52,10 @@ pub struct NetworkSettings {
     pub max_in_connection_overflow: usize,
     /// Max operations per message in the network to avoid sending to big data packet.
     pub max_operations_per_message: u32,
+    /// Read limitation for a connection in bit by 0.100 milliseconds
+    pub max_bit_read: u32,
```

Comment:
it's actually in bytes per second according to the doc https://github.com/adrien-zinger/async-speed-limit#usage